ci/cloudbuild/cloudbuild.yaml

```diff
@@ -38,7 +39,7 @@ steps:
     '--context=dir:///workspace/ci',
     '--dockerfile=ci/cloudbuild/${_DISTRO}.Dockerfile',
     '--cache=true',
-    '--destination=gcr.io/${PROJECT_ID}/google-cloud-cpp-cloudbuild-docker/${_DISTRO}-image:${BUILD_ID}',
+    '--destination=gcr.io/${PROJECT_ID}/${_IMAGE}:${BUILD_ID}',
```

Comment:
So the name of the image already includes the project?  Any reason to repeat it above?

Reply:
I think not. Changed to simply "cloudbuild".